cookbook/core/control_flow/run_conditions.py

```diff
@@ -222,11 +224,11 @@ def nested_conditions(my_input: float) -> float:
 # to be the subset of outputs that all then-nodes produce. In the following example, we call square() in one condition
 # and call double in another.
 @task
-def sum_diff(a: float, b: float) -> (float, float):
+def sum_diff(a: float, b: float) -> float:
```

Comment:
can we change the name and the comment tho?

Reply:
i am fixing it quickly